client/extensions/woocommerce/app/reviews/review-actions-bar.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const ReviewActionsBar = ( { review, currentStatus, translate } ) => {
```

Comment:
It's not clear the difference between review.status and currentStatus. Maybe a comment?